flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/ZooKeeperSubmittedJobGraphsStoreITCase.java

```diff
@@ -114,7 +112,7 @@ public void testPutAndRemoveJobGraph() throws Exception {
 			verifyJobGraphs(jobGraph, jobGraphs.recoverJobGraph(jobId));
 
 			// Update (same ID)
-			jobGraph = createSubmittedJobGraph(jobGraph.getJobId(), 1);
+			jobGraph = createSubmittedJobGraph(jobGraph.getJobId());
 			jobGraphs.putJobGraph(jobGraph);
 
 			// Verify updated
```

Comment:
`verifyJobGraphs` verifies JobID and JobName, if we don't change both of them, we should remove such verification. Otherwise we' better to check update.

Reply:
ok understood

Reply:
fixed